src/unity/schemas/OperatorAccountConvertRequest.yml

```diff
@@ -0,0 +1,7 @@
+properties:
+  contractStartDate:
+    type: string
+    format: date-time
+    description: the date the contract starts
```

Comment:
Do we need a separate schema defined for this request? If so, then the `description` above should be generic, and the comment about the date and format should go here instead.

Reply:
I'm not sure if we need it, but it seems consistent with how we spec'ed  the "set default account" endpoint, and it makes maintenance easier in case we allow more "conversion" types/data. Good call on the descriptions, I'll make the change.